vm/src/vm/runners/cairo_runner.rs

```diff
@@ -78,24 +78,34 @@ impl From<Vec<MaybeRelocatable>> for CairoArg {
 #[derive(Clone, Default, Debug, PartialEq)]
 pub struct RunResources {
     n_steps: usize,
+    consume_steps: bool,
 }
 
 impl RunResources {
     pub fn new(n_steps: usize) -> Self {
-        RunResources { n_steps }
+        RunResources {
+            n_steps,
+            consume_steps: true,
```

Comment:
Does this need to be a field?
Can't it just be a public method that returns `n_steps.is_positive()`